packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -293,7 +293,7 @@ class LocalHistoryEntry {
 
   /// Remove this entry from the history of its associated [LocalHistoryRoute].
   void remove() {
-    _owner.removeLocalHistoryEntry(this);
+    _owner?.removeLocalHistoryEntry(this);
```

Comment:
do you have a test for what led to this change?

Reply:
Unfortunately I don't know why I did this, and nothing appears broken without it.  Reverting that part.